README.md

```diff
@@ -76,14 +76,21 @@ The basic workings of this filter are done via the [`filters::FilterBase` API](h
     once. If this is true, the processing pipeline expects the pointcloud
     to have fields int32 index, float32 stamps, and float32 vp_x, vp_y
     and vp_z viewpoint positions. If one of these fields is missing,
-    computeMask() throws runtime exception.
+    computeMask() throws runtime exception. For the LaserScan version, this
+    parameter should be set to the sensor frame if all messages are coming
+    from the same sensor. This will save some some conversions (cpu cycles).
 - `frames/fixed` (`string`, default: `"base_link"`)
 
     The fixed frame. Usually base_link for stationary robots (or sensor
     frame if both robot and sensor are stationary). For mobile robots, it
     can be e.g. odom or map. Only needed for point-by-point scans.
+- `frames/sensor` (`string`, default `""`)
 
-- `frames/filtering` (`string`, default: default is `frames/fixed`)
+    Frame of the sensor. In LaserScan version, it has to match the
+    `frame_id` of the incoming scans. In PointCloud2 version, the data
+    can come in a different frame from `frames/sensor`. The frame_id from
+    the incoming message will be taken as sensor frame if this value is `""`.
+- `frames/filtering` (`string`, default: `frames/fixed`)
```

Comment:
It's not really clear to me from this description that it can be empty even for LaserScans.

Reply:
Could you update the description (to prevent a ping-pong conversation)?